modules/nf-core/freyja/demix/meta.yml

```diff
@@ -0,0 +1,57 @@
+name: "freyja_demix"
+description: specify the relative abundance of each known haplotype
+keywords:
+  - variants
+  - fasta
+  - deconvolution
+  - wastewater
+tools:
+  - "freyja":
+      description: "Freyja recovers relative lineage abundances from mixed SARS-CoV-2 samples and provides functionality to analyze lineage dynamics."
+      homepage: "https://github.com/andersen-lab/Freyja"
+      documentation: "https://github.com/andersen-lab/Freyja/wiki"
+      tool_dev_url: "https://github.com/andersen-lab/Freyja"
+      doi: "http://dx.doi.org/10.1038/s41586-022-05049-6"
+      licence: "['BSD-2-Clause']"
+
+input:
+  # Only when we have meta
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
```

Comment:
Can you also add all other metas and specify in their description to which file they belong?